domain/application/service/storage/directive.go

```diff
@@ -57,9 +58,28 @@ func (s *Service) GetApplicationStorageDirectives(
 	ctx context.Context,
 	uuid coreapplication.UUID,
 ) ([]application.StorageDirective, error) {
+	if uuid.Validate() != nil {
+		return nil, coreerrors.NotValid
+	}
 	return s.st.GetApplicationStorageDirectives(ctx, uuid)
 }
 
+// GetApplicationStorageInfo returns the storage directives set for an application,
+// keyed to the storage name. If the application does not have any storage
+// directives set then an empty result is returned.
+//
+// If the application does not exist, then a [applicationerrors.ApplicationNotFound]
+// error is returned.
```

Comment:
```suggestion
// The following errors may be expected:
// - [coreerrors.NotValid] when the supplied application uuid is not valid.
// - [applicationerrors.ApplicationNotFound] if the application does not exist.
```

Reply:
Ahh, I just usually prefer simple statements because the err is scoped but updated.